Source/WebKit/UIProcess/Network/NetworkProcessProxy.messages.in

```diff
@@ -85,6 +85,10 @@ messages -> NetworkProcessProxy LegacyReceiver {
     GetWindowSceneAndBundleIdentifierForPaymentPresentation(WebKit::WebPageProxyIdentifier webPageProxyIdentifier) -> (String sceneIdentifier, String bundleIdentifier)
 #endif
 
+#if ENABLE(APPLE_PAY_REMOTE_UI)
+    PaymentCoordinatorEmbeddingUserAgent(WebKit::WebPageProxyIdentifier webPageProxyIdentifier) -> (String userAgent) Synchronous
```

Comment:
Does this have to be done synchronously? Looking at the code, it seems like it could be rewritten to make it async.

Reply:
It doesn't have to be synchronous. I'd gone with this because the asynchronous version makes the call site a little verbose, but on second thought that's a tough sell.

Reply:
Done! https://github.com/WebKit/WebKit/commit/5f513a0a1ef25c698b67c93740b9a97ea9dcb822